src/utils.js

```diff
@@ -0,0 +1,7 @@
+export const buildDateRangeQuery = name => values => {
+  const [startDateString, endDateString] = values[0]?.split(':') || [];
+
+  if (!startDateString || !endDateString) return '';
```

Comment:
Let's do not use oneline "if" condition, in some modules we have lint rule for it, add brackets or return a ternary operator